api.go

```diff
@@ -545,6 +573,9 @@ type SearchOptions struct {
 	// it's up to the consumer of the result to remove those lines.
 	NumContextLines int
 
+	// TODO comments
+	ChunkMatches bool
```

Comment:
I suspect we want to move to only using ChunkMatches. Is that the case? If so, does this exist for the transition?